src/netlink_namespace.c

```diff
@@ -683,34 +682,59 @@ static int ir_config(const char *key, const char *value) {
 
 static int ir_init (void)
 {
-  DEBUG("Initializing netlink_namespace plugin\n");
-  if (ns_name != NULL) {
-    DEBUG("netlink_namespace plugin: ir_init: Switching to namespace on file '%s'.", *ns_name);
+  int selfns, targetns;
 
-    int fd = open(*ns_name, O_RDONLY);
-    if (fd  < 0) {
-        ERROR("Unable to open() %s", *ns_name);
-    }
-
-    if (setns(fd, CLONE_NEWNET) != 0) {
-       ERROR("netlink_namespace plugin: ir_init: Unable to setns() to %s.", *ns_name);
-       return (-1);
-    }
-  } else {
+  DEBUG("Initializing netlink_namespace plugin\n");
+  if (ns_name == NULL) {
     ERROR("netlink_namespace plugin: ir_init: Did not get the namespace name?");
     return (-1);
```

Comment:
I just mimic the same syntax as the rest of the file.
Anyway, will change it for the ones that I added.